tests/src/FunctionalJavascript/Form/AuthenticationFormJsTest.php

```diff
@@ -321,7 +317,7 @@ protected function validateForm(callable $visitFormAsAdmin): void {
     // Test invalid client id.
     $client_id = $this->randomGenerator->word(8);
     $page->fillField('Client ID', $client_id);
-    $this->assertSendRequestMessage('.messages--error', "Failed to connect to the OAuth authorization server. The given username ({$this->username}) or password or client ID ({$client_id}) or client secret is incorrect. Error message: ");
+    $this->assertSendRequestMessage('.messages--error', "Failed to connect to the OAuth authorization server. The given username ({$this->username}) or password or client ID ({$client_id}) or client secret is incorrect. Error message: {\"error\":\"unauthorized\",\"error_description\":\"Bad credentials\"}");
```

Comment:
@Jaesin I do not think that you would like to re-add these :) Issue caused by the rebase?